Sources/ApolloUtils/LinkedList.swift

```diff
@@ -0,0 +1,185 @@
+/// A doubly linked list implementation.
```

Comment:
l33t h4x0rz!!1! 

I'm kind of surprised there isn't anything in the stdlib or any of the helper libs around it - might be worth contributing?

Reply:
(Thinking specifically of [`swift-collections`](https://github.com/apple/swift-collections))

Reply:
IIRC there were optimizations in this implementation that made it specific to our performance needs.